Sources/Testing/Testing.docc/LimitingExecutionTime.md

```diff
@@ -0,0 +1,64 @@
+# Limiting the execution time of a test
+
+<!--
+This source file is part of the Swift.org open source project
+
+Copyright (c) 2023 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+Set limits on how long a test can run for until it fails.
+
+## Overview
+
+Some tests may naturally run slowly: they may require significant system
+resources to complete, may rely on downloaded data from a server, or may
+otherwise be dependent on external factors.
+
+If a test may hang indefinitely or may consume too many system resources to
+complete effectively, consider setting a time limit for it so that it will
+be marked as failing if it runs for an excessive amount of time. Use the
+``Trait/timeLimit(_:)`` trait as an upper bound:
+
+```swift
+@Test(.timeLimit(.seconds(60 * 60)) func serve100CustomersInOneHour() async {
+  for _ in 0 ..< 100 {
+    let customer = await Customer.next()
+    await customer.order()
+    ...
+  }
+}
+```
+
+If the above test function takes longer than 60 × 60 seconds (i.e. one hour) to
+execute, the task in which it is running will be [cancelled](https://developer.apple.com/documentation/swift/task/cancel())
+and the test will fail with an issue of kind
+``Issue/Kind-swift.enum/timeLimitExceeded(timeLimitComponents:)``.
+
+- Note: if multiple time limit traits apply to a test, the shortest time limit
```

Comment:
Not related to the content of this article, but on GitHub's PR review web UI I am seeing what appears to be code syntax highlighting in the remainder of this .md file, in paragraphs which don't have code. E.g. the words "for" and "is" are highlighted. Is there something invalid about the Markdown syntax? I wonder if it has gotten confused about the presence of the × multiplication character, does that need to be HTML encoded or something?

Reply:
Looks like the GitHub syntax highlighter's confused. It doesn't appear to affect output?

Reply:
Changing to `&times;` didn't help. 🤷